src/js/hooks/inboundV2/send/useInboundSendForm.js

```diff
@@ -171,6 +180,31 @@ const useInboundSendForm = ({ previous }) => {
     }
   };
 
+  const sendFiles = async () => {
+    const data = new FormData();
+
+    files.forEach((file, idx) => {
+      data.append(`filesContents[${idx}]`, file);
+    });
+
+    await stockMovementApi.uploadDocuments(stockMovementId, data);
+    if (files.length > 1) {
+      notification(NotificationType.SUCCESS)({
+        message: translate(
+          'react.stockMovement.alert.filesSuccess.label',
+          'Files uploaded successfully!',
+        ),
+      });
+      return;
+    }
+    notification(NotificationType.SUCCESS)({
+      message: translate(
+        'react.stockMovement.alert.fileSuccess.label',
+        'File uploaded successfully!',
+      ),
+    });
+  };
```

Comment:
What if uploadDocuments returns a fake 200? Then you will show information that the files are uploaded successfully, even if they are not.

Reply:
this should be wrapped in the try/finally also